app/export.go

```diff
@@ -161,7 +163,13 @@ func (app *App) prepForZeroHeightGenesis(ctx sdk.Context, jailAllowedAddrs []str
 	counter := int16(0)
 
 	for ; iter.Valid(); iter.Next() {
-		addr := sdk.ValAddress(iter.Key()[1:])
+		key := iter.Key()
```

Comment:
Can we rather call it "PrefixedAddress" instead key to kay it clearer? I add comment why using 2

Reply:
I don't mind changing it, but in my opinion,`keyPrefixLength` is more accurate. 
The key is the prefix + address, and we remove the key using the length.  

I added an answer here 
https://github.com/zeta-chain/node/pull/3033#discussion_r1822757877
Overall, we should have always used 2. It was just never caught cause we have never tried exporting for zero-height